classes/submission/reviewAssignment/ReviewAssignmentDAO.php

```diff
@@ -669,6 +669,41 @@ public function deleteByReviewRoundId($reviewRoundId): int
             ->map(fn ($reviewId) => $this->deleteById($reviewId))
             ->count();
     }
+
+    /**
+     * Delete all review assignments and review rounds for a given context ID.
+     *
+     * @param int $contextId
+     * @return void
+     */
+    public function deleteByContextId($contextId)
```

Comment:
For new code in `main`, use explicit typehinting:
```php
public function deleteByContextId(int $contextId) : void
```
And remove the redundant info from the self-documentation, which would then read just:
```
    /**
     * Delete all review assignments and review rounds for a given context ID.
     */
```